### ISSUE TITLE
Provide a task option to suppress an entire package

### ISSUE THREAD

**Issue by codeprogression** [Kotlin/dokka#128]
I have a package that I would like to prevent from generating into the javadocs/kdocs.

I added `@suppress` to all of the class files within the packages but the package still shows up in the javadoc/kdoc index and generates a package page with no methods.

It would be great to have a directive in the dokka tasks for excluding the package. 

```gradle
dokka {
  ...
  suppressPackages = ['com.example.internal.*', 'com.example.secret']
  ...
}
```

Something like that would allow me to generate external vs internal documentation.

Thanks!

**Comment by mikehowson** [comment]
Yes please!

**Comment by semoro** [comment]
Fixed in GH-222

**Comment by semoro** [comment]
Fixed in [0.9.16](https://github.com/Kotlin/dokka/releases/tag/0.9.16)
